### PR TITLE
Feature/ctsd synthesis

### PR DESCRIPTION
**Pull request by hammal**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -16,5 +16,5 @@
     "ylabel",
     "ylim"
   ],
-  "python.formatting.provider": "black"
+  "python.formatting.provider": "autopep8"
```
Comment (hammal) [inline .vscode/settings.json:19]: This change I'm not so sure It's a good idea.

Comment (fredrief) [reply .vscode/settings.json:19]: I think it was an accident from my side, good that you picked it up

**Comment by hammal** [comment]
alright, think I have the multi-level quantiser in place now

**Comment by hammal** [comment]
fixes #84

**Pull request merged**